plugins/woocommerce-admin/client/core-profiler/pages/Plugins.tsx

```diff
@@ -0,0 +1,210 @@
+/**
+ * External dependencies
+ */
+import { __, sprintf, _n } from '@wordpress/i18n';
+import { Button } from '@wordpress/components';
+import interpolateComponents from '@automattic/interpolate-components';
+import { Link } from '@woocommerce/components';
+import { Extension, ExtensionList } from '@woocommerce/data';
+import { useState } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { CoreProfilerStateMachineContext } from '../index';
+import { PluginsInstallationRequestedEvent, PluginsPageSkippedEvent } from '..';
+import { Heading } from '../components/heading/heading';
+import { Navigation } from '../components/navigation/navigation';
+import { PluginCard } from '../components/plugin-card/plugin-card';
+import { getAdminSetting } from '~/utils/admin-settings';
+
+const locale = ( getAdminSetting( 'locale' ) || 'en_US' ).replace( '_', '-' );
```

Comment:
Hey @moon0326 👋 I'm seeing error below in my local and JN site. 

```
react.js?ver=18.2.0:1397 Uncaught TypeError: ((0 , _utils_admin_settings__WEBPACK_IMPORTED_MODULE_8__.getAdminSetting)(...) || "en_US").replace is not a function
```

I guess it should be like this?

```suggestion
const locale = ( getAdminSetting( 'locale' ).siteLocale || 'en_US' ).replace(
	'_',
	'-'
);
```

Reply:
Thank you! Yup. I've added `?` just in case in https://github.com/woocommerce/woocommerce/pull/38405/commits/34fd07f10f240ee22958afa00df24ccd3747d407